Handler/ExcitationHandler.cpp

```diff
@@ -20,7 +21,7 @@
 
 #include "ExcitationHandler.h"
 
-static const size_t evaporation_threshold = 1e3;
+static const size_t evaporation_threshold = 1e3 * CLHEP::keV;
```

Comment:
Why 1e3 keV? This is too high energy for the evaporation threshold.

Reply:
It was iteration limiter all along...